### PR TITLE
Command-line interface

### LINKED ISSUE #5: Command-line interface
Create a CLI for the major functionality of this package and its sister packages:

- Convert raw flow graph to semantic flow graph
- Convert flow graphs to Graphviz and TikZ for visualization
- Convert Python code to raw flow graph (via [pyflowgraph](https://github.com/ibm/pyflowgraph) and [PyCall.jl](https://github.com/JuliaPy/PyCall.jl))
- Convert R code to raw flow graph (via [rflowgraph](https://github.com/ibm/rflowgraph) and [RCall.jl](https://github.com/JuliaInterop/RCall.jl))

### PR DESCRIPTION
**Pull request by epatters**
Getting this started. I should add a little documentation in the README.

Closes #5.

### CODE REVIEW

File: src/CLI.jl [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+import Catlab.Diagram: Graphviz
+using SemanticFlowGraphs
+
+# CLI arguments
+###############
+
+const settings = ArgParseSettings()
+
+@add_arg_table settings begin
+  "record"
```
[the remark this answers is not in the document]
Comment (epatters) [reply src/CLI.jl]: Already supported! To use this feature, you need to have both PyCall.jl and pyflowgraph installed. I will document this.

File: src/CLI.jl [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+    help = "convert raw flow graph to semantic flow graph"
+    action = :command
+  "visualize"
+    help = "visualize flow graph"
+    action = :command
+end
+
+@add_arg_table settings["record"] begin
+  "path"
+    help = "input file or directory"
```
[the remark this answers is not in the document]
Comment (epatters) [reply src/CLI.jl]: Good point.

File: src/CLI.jl (hunk 1 above)
[the remark this answers is not in the document]
Comment (ioana-blue) [reply src/CLI.jl]: sounds good. specially the "document this" part :)

**Comment by epatters** [comment]
Whoops. I meant to merge the Python CLI PR but I merged this one instead.

**Pull request merged**